moped-editor/src/views/projects/projectView/ProjectSummary.js

```diff
@@ -94,14 +96,24 @@ const ProjectSummary = () => {
   return (
     <CardContent>
       <Grid container spacing={2}>
-        {projectDetails.map(detail => (
-          <Grid item xs={6}>
-            <Box mb={2}>
-              <h4>{detail.label}</h4>
-              <p>{formatValue(detail)}</p>
-            </Box>
+        <Grid item xs={6}>
+          <Grid container>
+            {projectDetails.map(detail => (
+              <Grid item xs={6}>
+                <Box mb={2}>
+                  <h4>{detail.label}</h4>
+                  <p>{formatValue(detail)}</p>
+                </Box>
+              </Grid>
+            ))}
           </Grid>
-        ))}
+        </Grid>
+        <Grid item xs={6}>
```

Comment:
I think we need to maybe update these side-by-side components to stack on smaller screen widths. maybe xs=12 and sm or md=6?

Reply:
This is a really good point, and I'll update before merge so we keep these fields and map looking sharp on smaller screens!